### PR TITLE
HDDS-10229. Fixes for dashboards

### PR DESCRIPTION
**Pull request by kerneltime**
## What changes were proposed in this pull request?
There are some fixes needed for various dashboards published.
Details are in the linked jira.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-10229

## How was this patch tested?

Dashboard was imported to my dev cluster and validated against load generation.

### CODE REVIEW

File: hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/reconciliation.sdt
Comment (adoroszlai) [inline hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/reconciliation.sdt:1]: I think this empty file is unintentionally added to the PR.

Comment (kerneltime) [reply hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/reconciliation.sdt:1]: Thank you.

File: hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json [hunk 1]
```diff
@@ -77,7 +69,7 @@
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "${DS_PROMETHEUS-1}"
+        "uid": "${DS_PROMETHEUS}"
```
Comment (tanvipenumudy) [inline hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json:72]: `Datasource ${DS_PROMETHEUS} was not found`

I have checked out the patch locally and tried viewing the dashboards using Graphana via a docker-compose cluster with the monitoring add-ons, the variable: `DS_PROMETHEUS` does not get resolved automatically - not sure if it has something to do with my local settings! (please find the image attached).

Dashboards Error

I tried removing the `uid` field under `datasource` after which I do not run into this issue - I get to see the graphs as expected.
```
"datasource": {
  "type": "prometheus"
},
```

**Review by tanvipenumudy — COMMENTED** [review]
Thank you for the patch!

File: hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json (hunk 1 above)
Comment (kerneltime) [reply hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json:72]: Did you import it as a new dashboard?

Comment (tanvipenumudy) [reply hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json:72]: While adding a new dashboard (by importing the `.json` file) and selecting the Prometheus source, I can see the graphs. However, when attempting to view graphs for the existing dashboards, I encounter the error message: `Datasource ${DS_PROMETHEUS} was not found`.

Comment (kerneltime) [reply hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json:72]: I was able to reproduce the issue. It only impacts the pre provisioned Dashboards in the dev environment but should be ok for any customer importing a dashboard. I will update with the fix that makes it work for both.

Comment (kerneltime) [reply hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json]: Done

**Review by duongkame — APPROVED** [review]
Thanks for the patch @kerneltime . LGTM.

**Comment by adoroszlai** [comment]
Thanks @kerneltime for the patch, @duongkame, @tanvipenumudy for the review.

File: hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json (hunk 1 above)
Comment (PhilippeLan) [reply hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json]: Dear ozone comiters, 
The latest Grafana is depreciating certain visual currently used within the current object-metrics dashboard.
I've updated upon recent expectation hope this helps.

[Ozone  - Object Metrics update dahsboards.json](https://github.com/apache/ozone/files/14723282/Ozone.-.Object.Metrics.update.dahsboards.json)

Comment (kerneltime) [reply hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - ListKey Metrics.json]: Thanks @PhilippeLan will include these changes.

**Pull request merged**